languages/python/oso/tests/test_polar_roles.py

```diff
@@ -2673,8 +2670,8 @@ def test_roles_integration(init_oso, sample_data):
     assert not oso.is_allowed(leina, "edit", ios_laggy)
     assert not oso.is_allowed(steve, "edit", ios_laggy)
 
-    oso.actor = leina
-    oso.checked_permissions = {Repo: "pull"}
+    # oso.actor = leina
+    # oso.checked_permissions = {Repo: "pull"}
     # auth_session = auth_sessionmaker()
```

Comment:
Can we delete all this commented code?

Reply:
Yeah I s'pose; was letting it hang around in the hopes we'd get to testing partial eval soon, but can always pull it out of the Git woodpile when we're ready to do that.

Reply:
actually gonna avoid doing it for now b/c I think it'll cause @saolsen some merge conflict headaches on #969. will clean it up as part of that PR or as follow-up.